cluster/src/main/java/com/hartwig/pipeline/datatypes/FileTypes.java

```diff
@@ -11,6 +11,8 @@
     public static final String CRAI = ".crai";
     public static final String CRAM = ".cram";
 
+    public static final String TSV = "tsv";
```

Comment:
No leading '.' required?

Reply:
Nope because the extension is not used an "extension upon extension" such as TBI, BAI etc.

Actually ".cram" is the inconsistent one in this list! Will correct it